apps/docs/components/Image/Image.tsx

```diff
@@ -0,0 +1,14 @@
+import Image, { type ImageProps } from "next/image";
+
+import "./image.css";
+
+interface NextImageProps extends Omit<ImageProps, "src" | "alt" | "width" | "height"> {
+    alt: string;
+    src: string;
+    width: number;
+    height: number;
+}
+
+export const NextImage: React.FC<NextImageProps> = ({ alt, src, width, height, ...props }) => {
```

Comment:
Same comment as for the Card component
```suggestion
const NextImage = ({ alt, src, width, height, ...props }: React.PropsWithoutRef<NextImageProps>) => {
```